packages/devtools-network-console/src/ui/ResponseViewer/CookiesTable.tsx

```diff
@@ -27,26 +28,61 @@ const dataCols: DataGridColumn[] = [
         columnDataKey: 'name',
         title: 'Name',
         columnWidth: '20%',
+        header: (config: DataGridHeaderRenderConfig) => {
+            return (
+                <div className={config.classNames} role="columnheader" key={config.key} style={{gridColumn: '1 / auto', textAlign: 'center'}}>
+                    <LocText textKey={`CookiesTable.Header.${config.title}`} />
+                </div>
+            );
```

Comment:
nit: do you think it's worth creating a header component where the column for style and the other params can be passed in?

Reply:
Did it, thanks